src/sinks/datadog/metrics/config.rs

```diff
@@ -42,9 +42,9 @@ pub const MAXIMUM_PAYLOAD_SIZE: usize = 62_914_560;
 pub struct DatadogMetricsDefaultBatchSettings;
 
 impl SinkBatchSettings for DatadogMetricsDefaultBatchSettings {
-    const MAX_EVENTS: Option<usize> = Some(100_000);
+    const MAX_EVENTS: Option<usize> = Some(20);
     const MAX_BYTES: Option<usize> = None;
-    const TIMEOUT_SECS: NonZeroU64 = unsafe { NonZeroU64::new_unchecked(2) };
+    const TIMEOUT_SECS: NonZeroU64 = unsafe { NonZeroU64::new_unchecked(1) };
```

Comment:
This seemed different enough that I assumed it was mistakenly updated.

Reply:
I assume you're saying that statement in the context of what the actual documentation for the sink says?

The calculation for 100K events/2 seconds was based on a timeliness/batch efficiency trade-off to ensure the default was able to saturate throughput as much as possible.  I feel like 20 events/1 second is still too low of a default, so this might be a case where we could change to something lower than the upper bound (like I did) but higher than the documented defaults, which are just.... staggeringly low.

Maybe 1000 events/1 second?

Reply:
Ah, yeah, I see. This was based on what the default was prior to the rewrite, but if you tested it with these new values I'm happy to update them to that.

Reply:
Updated in 2ce5b5c8e